index.js

```diff
@@ -75,5 +76,31 @@ function handleLine(line) {
         }
     } else if (line.indexOf("/reload-partitions") === 0) {
         sv.reloadPartitionMap();
+    } else if (line.indexOf("/globalban") === 0) {
+        var args = line.split(" "); args.shift();
+        if (args.length === 2 && net.isIPv4(args[0]) || net.isIPv6(args[0])) {
```

Comment:
Operator precedence means that this is parsed as `(args.length === 2 && net.isIPv4(args[0])) || net.isIPv6(args[0])`, also if you want to support both you can just check if `net.isIP()` is nonzero.